packages/react-devtools-shared/src/backend/views/TraceUpdates/canvas.js

```diff
@@ -43,8 +43,30 @@ function drawNative(nodeToData: Map<HostInstance, Data>, agent: Agent) {
 }
 
 function drawWeb(nodeToData: Map<HostInstance, Data>) {
+  // if there are no nodes to draw, detach from top layer
+  if (nodeToData.size === 0) {
+    if (canvas !== null) {
+      try {
+        if (canvas.matches(':popover-open')) {
+          // $FlowFixMe[prop-missing]
+          // $FlowFixMe[incompatible-use]
+          canvas.hidePopover();
```

Comment:
Why it needs to be hidden in this case?

Reply:
If we don't remove it from the toplayer, it would be pushed behind when other toplayer elements appear later.